### PR TITLE
Turkiye Fetch is born

### PR DESCRIPTION
**Pull request by majesticio**
Added adapters/turkiye.js and updated sources/tr.json config file

source url: https://sim.csb.gov.tr/Services/GetAirQualityStations?type=0

sim_site_screenshot

### CODE REVIEW

File: adapters/turkiye.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+ 
+ /**
+  * Fetches the data for a given source and returns an appropriate object
+  * @param {object} source A valid source object
+  * @param {function} cb A callback of the form cb(err, data)
+  */
+  
+ exports.fetchData = function (source, cb) {
+ 
+     const ALL_DATA =
```
Comment (russbiggs) [inline adapters/turkiye.js:26]: This should be coming from the source JSON. e.g. https://github.com/openaq/openaq-fetch/blob/master/adapters/airnow-http.js#L73

File: sources/tr.json [hunk 2]
```diff
@@ -11,5 +11,18 @@
             "info@openaq.org"
         ],
         "active": true
+    },
+    {
+        "url": "https://sim.csb.gov.tr/SERVICES/airquality",
```
Comment (russbiggs) [inline sources/tr.json:16]: This should be what you have in sourceURL and sourceURL should be this

File: sources/tr.json [hunk 3]
```diff
@@ -11,5 +11,18 @@
...
+        "adapter": "turkiye",
+        "name": "Turkey",
```
Comment (russbiggs) [inline sources/tr.json:18]: lets change name to "Turkiye"

**Review by russbiggs — CHANGES_REQUESTED** [review]

File: adapters/turkiye.js (hunk 1 above)
Comment (majesticio) [reply adapters/turkiye.js:26]: so I can replace ALL_DATA with source.url?

Comment (russbiggs) [reply adapters/turkiye.js]: yeahj

**Review by russbiggs — CHANGES_REQUESTED** [review]

**Review by russbiggs — APPROVED** [review]

**Pull request merged**